src/test/java/sorinfoca/Tests/Tema8/TestPage/TestPages.java

```diff
@@ -0,0 +1,57 @@
+package sorinfoca.Tests.Tema8.TestPage;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import sorinfoca.Tests.Tema8.pages.*;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class TestPages extends BaseTest {
+
+    private static WebDriver driver;
```

Comment:
You don't need these two lines here, we're going to use driver from BaseTest class whenever needed. Wait will be most probably used in PageObject classes. Please remove these two lines from here. Thanks!